format.sh

```diff
@@ -0,0 +1,48 @@
+is_source_file() {
```

Comment:
Can you add at least a small description of what the script does. What if formats, if it takes arguments, and what files will ignore? Just to have a general idea of why this script exist.